shareMethod.sh

```diff
@@ -9,9 +9,9 @@ execWithTimeout() {
         echo "execWithTimeout() Command not specified"
         exit 2
     fi
-    local cmd="/bin/sh -c \"$1\""
+    export cmd="/bin/sh -c \"$1\""
```

Comment:
The `expect << EOF` can not recognize the local variable.  I test several ways, and it is a unique way to do so.